contracts/GaugeController.sol

```diff
@@ -33,16 +79,43 @@ contract GaugeController is IGaugeController {
         rewardVault = _rewardVault;
     }
 
+    modifier requireGauge(address _gauge) {
+        require(isGauge(_gauge), "GaugeController:invalid-address");
+        _;
+    }
+
+    /* ================================================================================ */
+    /* External Functions                                                               */
+    /* ================================================================================ */
+
+    function isGauge(address _gauge) public view returns (bool) {
+        return gaugeScaleTwabs[_gauge].details.balance > 0;
+    }
+
+    /**
+     * @notice Deposit tokens in GaugeController and increase User balance.
+     * @param _to  Receivzer of the deposited tokens
+     * @param _amount  Amount of tokens to be deposited
+    */
     function deposit(address _to, uint256 _amount) public {
         balances[_to] += _amount;
```

Comment:
We should add requires to check that we are not deposition to address zero and also that the amount is greater than zero.